spec/lib/api/redfish-1.0/event-service-spec.js

```diff
@@ -39,62 +39,41 @@ describe('Redfish Event Service', function () {
             }
         }]
     };
-    
-    before('start HTTP server', function () {
-        this.timeout(5000);
+
+    before(function() {
```

Comment:
Originally, this is to stub `redfishTool` before httpServer is started, so we can't combine these two `before` section.